zbsprimarystorage/zbsprimarystorage/zbsagent.py

```
@@ -229,23 +245,27 @@ def sync_metadata(self, req):
             return jsonobject.dumps(rsp)
 
         return jsonobject.dumps(rsp)
 
     @replyerror
     def ping(self, req):
+        cmd = jsonobject.loads(req[http.REQUEST_BODY])
         rsp = AgentResponse()
 
+        current_cluster_uuid = zbsutils.get_cluster_uuid(cmd.clusterInfo.version)
+        if current_cluster_uuid != cmd.clusterInfo.uuid:
+            raise Exception('cluster uuid does not match, current cluster uuid[%s], old cluster uuid[%s]' % (current_cluster_uuid, cmd.clusterInfo.uuid))
```

Ruff (0.12.2)

Create your own exception

(TRY002)